client/state/selectors/get-reader-aliased-follow-feed-url.js

```diff
@@ -21,11 +23,18 @@ import { prepareComparableUrl } from 'state/reader/follows/utils';
 export default function getReaderAliasedFollowFeedUrl( state, feedUrl ) {
 	const urlKey = prepareComparableUrl( feedUrl );
 
+	// first check for exact match
 	if ( state.reader.follows.items[ urlKey ] ) {
 		return urlKey;
 	}
 
-	const foundAlias = find( state.reader.follows.items, f => includes( f.alias_feed_URLs, urlKey ) );
+	// then check if any follows have saved aliases OR if there is a matching autodiscoverable alias
+	const foundAlias = find(
+		state.reader.follows.items,
+		( follow, key ) =>
+			includes( follow.alias_feed_URLs, urlKey ) ||
+			some( commonExtensions, ext => `${ urlKey }/${ ext }` === key )
```

Comment:
If I'm already following a feed with a common extension (e.g. http://jancavan.com/rss), I don't think this'll match anything else. Should we try stripping the last part off the URL  (i.e. `/rss` in this case) too?